### ISSUE TITLE
Ensure text and images of text provide sufficient contrast

### ISSUE THREAD

**Issue by blnkt** [lsst-epo/rubin-obs-client#72]
## Description
There must be sufficient contrast between foreground and background text colors and images of text (including the hover and focus state of text and images of text). Sufficient contrast ensures that people with low vision, people who are color blind, users viewing the page without color, and users of monochrome screens can understand page content.

### Header
- [X] Language Toggle "disabled language" indicator text against its background should have a minimum contrast of 4.5:1.

[This is a dropbox link with updated design for Language Toggle colors](https://www.dropbox.com/s/3tnvsph1uk9h6q9/20220304%20language%20toggle.pdf?dl=0).  The current background color is the existing scss var `turquoise85`.  The new background color is `#00BABC`, which is extremely close to `turquoise50`.  Unless there's an impediment I'm overlooking, please make `turquoise50: #00BABC` and transition this toggle background-color from `turquoise85` to `turquoise50`.

![10128f32f27066dba5c7cd41db49f5930023fd1f-1641470810](https://user-images.githubusercontent.com/8166060/153084772-801a5a0f-7348-4049-ab05-9b7cb20fa805.png)

### Buttons
- [x] "White text against red background" button should have a minimum contrast of 4.5:1.  Either need a darker red, larger text, or both.

![aa86262315af5ee90ffcfc8872811b63a051de36-1641272517](https://user-images.githubusercontent.com/8166060/153085496-f1d0f6cc-ee4e-47b3-9d87-90554c4f46d4.png)

**New Design**
[This is a dropbox link with updated design](https://www.dropbox.com/s/5puxqa5nue2lcdl/20220310%20Accessibility%20updates%20to%20buttons.pdf?dl=0).  This design is for changes to the hover/focus states, but the new colors are updated too.  New red is `#cf4040` and should be used for the value of the `red` scss variable.

- [x]  "White text against dark green background" button should have a minimum contrast of 4.5:1.  Either need a darker green, larger text, or both.  

![Screen Shot 2022-02-08 at 2 33 24 PM](https://user-images.githubusercontent.com/8166060/153086916-6fc13cb9-dc71-4215-8972-8c8aed40e24b.png)

**New Design**
[This is a dropbox link with updated design](https://www.dropbox.com/s/5puxqa5nue2lcdl/20220310%20Accessibility%20updates%20to%20buttons.pdf?dl=0).  This design is for changes to the hover/focus states, but the new colors are updated too.  New green indicated in the design is `#12726c`, which is extremely similar turquoise85, so please just make turquoise85: `#12726c` and use that variable here.

### CTA Grid
- [x] "White text against dark green background" grid item titles should have a minimum contrast of 4.5:1.  Either need a darker green, larger text, or both.

![94a3fc4d050f35475f4188f5b795a6878a129622-1640950740](https://user-images.githubusercontent.com/8166060/153086347-65528a08-ae17-46ce-89cd-46c2c0c94c75.png)

**New Design**
[This is a dropbox link with updated design for CTA Grid Item title background color](https://www.dropbox.com/s/fd70j7zcwoz54ci/20220223%20CTA%20Grid.pdf?dl=0).  The current background color is the existing scss var `turquoise55`.  The new background color is the existing scss var `turquoise85`.  Unless there's an impediment I'm overlooking, please transition all uses of `turquoise55` to `turquoise85`, and remove the `turquoise55` var altogether.

### Pagination
- [x] Green text for page numbers and "next" should have a minimum contrast of 4.5:1.  Either need a darker green, larger text, or both.

![Screen Shot 2022-02-08 at 2 35 41 PM](https://user-images.githubusercontent.com/8166060/153087528-793056d3-8d12-414e-bce5-012b85e7b3d1.png)

**New Design**
[This is a dropbox link with updated design for Pagination link colors](https://www.dropbox.com/s/wy56rlz1dhbw1yc/20220304%20pagination%20links.pdf?dl=0).  The current color is the existing scss var `turquoise55`.  The new background color is the existing scss var `turquoise85`.  Unless there's an impediment I'm overlooking, please transition all uses of `turquoise55` to `turquoise85`, and remove the `turquoise55` var altogether.

### Breadcrumb
- [x] Green active link in breadcrumb should have a minimum contrast of 4.5:1.  Either need a darker green, larger text, or both.

![656fe8e523c6b732af736be6e6ab7f11fda064fc-1641293483](https://user-images.githubusercontent.com/8166060/153087722-0cadcb2b-1921-4105-be4e-cc89a93063c3.png)

**New Design**
[This is a dropbox link with updated design for breadcrumb links](https://www.dropbox.com/s/it2m5udde6hdyay/20220325%20Breadcrumb%20styles.pdf?dl=0)

## CIC Estimate
1 point

**Comment by blnkt** [comment]
For: **CTA Grid**

[This is a dropbox link with updated design for CTA Grid Item title background color](https://www.dropbox.com/s/fd70j7zcwoz54ci/20220223%20CTA%20Grid.pdf?dl=0)

The current background color is the existing scss var `turquoise55`.  The new background color is the existing scss var `turquoise85`.  Unless there's an impediment I'm overlooking, please transition all uses of `turquoise55` to `turquoise85`, and remove the `turquoise55` var altogether.

**Comment by blnkt** [comment]
This issue is no longer blocked by design. All new designs are provided through dropbox links.
